### PR TITLE
quandl: init at 3.0.0

### PR DESCRIPTION
**Pull request by NikolaMandic**
###### Things done
- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [ ] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x ] Fits [CONTRIBUTING.md]  
- [x] tested by importing in shell
  (https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -27407,4 +27407,33 @@ in modules // {
...
+     src = pkgs.fetchurl {
+       url= "mirror://pypi/q/quandl/${name}.tar.gz";
+       sha256 = "d4e698eb39291e0b281975813054101f3dfb379dead10d34d7b536e1aad60584";
+     };
+
+     propagatedBuildInputs = with self; [
+	numpy
+	ndg-httpsclient
+	dateutil
+	inflection
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: can't find this dependency

Comment (NikolaMandic) [inline pkgs/top-level/python-packages.nix]: it is in another pull req

Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: ok, if you have packages that depend on each other, then put them in one PR but separate commits

Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: also, please write `pythonPackages.<name package>` since that makes it easier to spot which changes are python related.

Comment (NikolaMandic) [inline pkgs/top-level/python-packages.nix]: hm what do you mean with this last one?
can the separate pulls pass this time?

**Comment by NikolaMandic** [comment]
@FRidh changed now. but where do you want pythonPackages to be put

**Comment by FRidh** [comment]
@NikolaMandic commit name. Check for example `git log --grep python`  and you will see many Python packages.

**Comment by NikolaMandic** [comment]
@FRidh done

**Comment by FRidh** [comment]
pushed in 2df46320048f834f3434d989c653a5c02b744807 and 31bcdeea80ad90886578f17a5e68eb874606d26f

**Comment by NikolaMandic** [comment]
@FRidh why did you close this? you told me to put it in one pull with separate commits and different commit message

**Comment by FRidh** [comment]
@NikolaMandic 
> FRidh commented a day ago
>
> pushed in 2df4632 and 31bcdee

**Comment by NikolaMandic** [comment]
ok got it. you pushed it from somewhere else. thanks
